package.json

```diff
@@ -37,6 +37,12 @@
     "test-update-snapshot": "lerna run --scope web-registry test-update-snapshot --stream"
   },
   "devDependencies": {
+    "@graphql-codegen/cli": "1.21.4",
+    "@graphql-codegen/introspection": "1.18.2",
+    "@graphql-codegen/typescript": "1.22.0",
+    "@graphql-codegen/typescript-graphql-files-modules": "1.18.1",
+    "@graphql-codegen/typescript-operations": "^1.18.0",
+    "@graphql-codegen/typescript-react-apollo": "2.2.4",
```

Comment:
I had to move them to the top level `package.json`, graphql codegen was failing because it couldn't find the @graphql-codegen plugins (see https://www.graphql-code-generator.com/docs/getting-started/installation#monorepo-project) cc/ @haveanicedavid 
One better option would probably be to use nohoist https://github.com/dotansimha/graphql-code-generator/issues/3118 but this might require more work for it to work with the different versions of graphql we have and this was out of scope here

Reply:
I think this makes sense @blushi - should we make an issue to try and use nohoist?

Reply:
done: https://github.com/regen-network/regen-web/issues/852